src/servers/ZoneServer2016/managers/smeltingmanager.ts

```diff
@@ -348,17 +346,6 @@ export class SmeltingManager {
         1,
         false
       );
-      server.sendDataToAllWithSpawnedEntity<CharacterPlayWorldCompositeEffect>(
```

Comment:
why removing this ?

Reply:
Oh I was testing something related to the sound playing on bee boxes and forgot to revert this.